packages/wdio-cli/src/config.js

```diff
@@ -161,13 +161,13 @@ export const QUESTIONNAIRE = [{
     message: 'Where is your automation backend located?',
     choices: [
         'On my local machine',
-        'In the cloud using Sauce Labs, Browserstack or Testingbot',
-        'In the cloud using a different service',
+        'In the cloud using Sauce Labs',
+        'In the cloud using Browserstack or Testingbot or a different service',
```

Comment:
I'd prefer the following as it doesn't begin with specific services but others in general first:
`In the cloud using a different service like Browserstack or Testingbot`